app/components/lazy-minted/LazyMinted.tsx

```
@@ -0,0 +1,117 @@
+              <th className="border border-slate-600 px-4 py-1 ">ID</th>
+              <th className="border border-slate-600 px-4 py-1">Media</th>
+              <th className="border border-slate-600 px-4 py-1">Name</th>
+              <th className="border border-slate-600 px-4 py-1">Description</th>
+              <th className="border border-slate-600 px-4 py-1">Supply</th>
+              <th className="border border-slate-600 px-4 py-1">More</th>
+            </tr>
+          </thead>
+          <tbody>
+            {nfts.map((nft, i) => (
+              <tr
+                key={nft.metadata.name + '-' + nft.id}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)